_sections/30-projects.md

```diff
@@ -17,3 +17,4 @@ style: center
 - [An Introduction to Applied Bioinformatics](http://readiab.org)
 - [QIIME](http://qiime.org)
 - [cual-id](https://github.com/johnchase/cual-id)
+- [gala](https://gala.readthedocs.io)
```

Comment:
Using Github stars as an indicator of prominence, you can go up one space to between QIIME and cual-id, if you like.